src/components/tools/tool-tile.tsx

```diff
@@ -197,6 +218,12 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
                             <DragTileButton divRef={elt => this.dragElement = elt}
                               hovered={hoverTile} selected={isTileSelected}
                               onClick={e => ui.setSelectedTile(model, {append: hasSelectionModifier(e)})} />;
+    const resizeTileButton = isDraggable &&
```

Comment:
```suggestion
    const resizeTileButton = isUserResizable &&
```
Right logic (testing here rather than down below), wrong test condition

Reply:
Oops